addOns/alertFilters/src/main/resources/org/zaproxy/zap/extension/alertFilters/resources/Messages.properties

```diff
@@ -70,7 +70,7 @@ alertFilters.dialog.add.label.scope.global		= Global
 alertFilters.dialog.button.apply	= Apply
 alertFilters.dialog.button.test		= Test
 alertFilters.dialog.filter.state.notapplied = Not applied
-alertFilters.dialog.filter.state.nottested = Not tested
+alertFilters.dialog.filter.state.untested = Not tested
```

Comment:
What's the problem here?

Reply:
They're both fine, probably more of a style choice.

Reply:
But why change the key and not the text? Better just fix typos IMO.

Reply:
Fwiw, `untested` is a word whereas `not tested` is really two words that one would naturally include some punctuation between.

Note that I'm not attached to the change, but that's the reason for proposing it.

Reply:
Like `notapplied` (and a thousand more), which was not changed (not asking for it either).